wadm/test/fixtures/mixed_spreads.yaml

```diff
@@ -0,0 +1,36 @@
+apiVersion: core.oam.dev/v1beta1
+kind: Application
+metadata:
+  name: my-example-app
+  annotations:
+    version: v0.0.1
+    description: "This is my app"
+spec:
+  components:
+    - name: userinfo
+      type: actor
+      properties:
+        image: wasmcloud.azurecr.io/fake:1
+      traits:
+        - type: spreadscaler
+          properties:
+            replicas: 4
+            spread:
+              - name: allhosts
+                weight: 67
+              - name: westcoast
+                requirements:
+                  zone: us-west-1
+                weight: 33
+        - type: linkdef
+          properties:
+            target: webcap
+            values:
+              port: 8080
+
+    - name: webcap
+      type: capability
+      properties:
+        contract: wasmcloud:httpserver
+        image: wasmcloud.azurecr.io/httpserver:0.13.1
```

Comment:
What are we, in the stone ages here?

Reply:
Whoops. Better go update the examples code I copied this from

Reply:
Actually this doesn't matter. It is a fixture 🙃

Reply:
Yeah I was just jesting :)